src/clang.rs

```diff
@@ -34,39 +48,67 @@ impl Cursor {
         }
     }
 
+    /// Is this cursor's referent a declaration?
     pub fn is_declaration(&self) -> bool {
         unsafe { clang_isDeclaration(self.kind()) != 0 }
     }
 
+    /// Get the null cursor, which has no referent.
     pub fn null() -> Self {
         Cursor { x: unsafe { clang_getNullCursor() } }
     }
 
-    // common
+    /// Get this cursor's referent's common name.
```

Comment:
I don't think you can describe this better than "Get this cursor's spelling", because it varies a lot depending on the cursor kind.